articles/Document Information Extraction with Large Language Models/README.md

```diff
@@ -0,0 +1,143 @@
+---
+title: "Document Information Extraction with Large Language Models (framework, methods,  ensemble, and evaluation)"
+author: ["Carvin Li / ML Engineer", "Anjing Wang / NLP, ML Engineer"]
+createTime: 2024-05-21
+tags: ["Information Extraction", "Large Language Model"]
+thumb: ""
```

Comment:
Could you add the `thumb` and `thumb_h` and replace the images in the document  as the design? Thanks
https://www.figma.com/design/GHDhmnzRQTAUdjNTgOPJmF/57Blocks-Website-V3?node-id=14042-2730&t=XfpMSvxqOyvotTWG-0

Reply:
Sure~